services/ui-src/src/forms/mcpar/cbss/cbss.schema.ts

```diff
@@ -1,3 +1,8 @@
-import { object } from "yup";
+import { object, string } from "yup";
 
-export default object({});
+export default object({
+  "cbss-1": string(),
+  "cbss-2": string(),
+  "cbss-3": string(),
+  "cbss-4": string(),
+});
```

Comment:
You'll want to always use our custom field validation schema (found in utils/forms/schemas) for these since the default Yup `string` just checks that the entered data is a string type, while our custom schema (built on top of Yup's schemas) adds all the more advanced checks. Every type of field validation that's needed in the app _should_ already exist in there. 😄

Reply:
Example below because GitHub is not doing what I want lol